flux-cli/src/main/java/com/marklogic/flux/api/SplitterOptions.java

```diff
@@ -50,7 +50,7 @@ public interface SplitterOptions {
 
     SplitterOptions outputClassifierPort(String classifierPort);
 
-    SplitterOptions outputClassifierProtocol(String classifierProtocol);
+    SplitterOptions outputClassifierProtocol(boolean classifierProtocol);
```

Comment:
Since this is now a boolean and no longer a protocol value, call it `classifierHttps` (and remove "output" as a prefix from all of these options - I don't know why I chose that for the sidecar-specific ones instead of "sidecar", it may have had something to do with these options being reused in the "copy" operation where everything has an "output" prefix).